### DISCUSSION TITLE
Data proxy: Connecting to MySQL that does not speak TLS

### DISCUSSION THREAD

**Discussion by konuch** [prisma/prisma#16493] [Q&A]
Hello guys, I want to connect my data proxy instance to a hosted MySQL server, but I'm getting following error:
```
Error opening a TLS connection: 
error:1408F10B:SSL
routines:ssl3_get_record:wrong version
number:../ssl/record/ssl3_record.c:331:
```
My connection string structure I pass to data proxy settings is following: `mysql://<USER:<PW>@<HOST_IP>:3306/<DB_NAME>`

Do you have an idea, how to solve this issue?

Thanks,
Rado

**Comment by nurul3101** [comment]
Hello @konuch 👋 

Are you passing any SSL-specific parameters in your connection string?

Also, the best way to get support for Data Proxy or Prisma Data Platform would be to open a support ticket from https://cloud.prisma.io/

**Reply by konuch** [reply]
Hello @nurul3101, thanks for you answer. No, I'm not passing any SSL parameters in, I'm using the same connection string that works when I try to connect to it from my local machine using PrismaClient without data proxy. So I expected the same string to work when using with Data Proxy. Are there any connection string parameters, that could tell the data proxy that target MySQL DB server does not support SSL connection? Thanks

**Reply by nurul3101** [reply]
I see you opened a support ticket, we can continue the conversation there.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
